paddle/framework/operator.h

```diff
@@ -83,7 +83,7 @@ class OperatorBase {
     return boost::get<T>(attrs_.at(name));
   }
 
-  virtual std::string DebugString() const;
+  virtual void DebugPrint(std::ostream* os) const;
```

Comment:
@wangkuiyi 
Because `DebugPrint` is a virtual method because Simple `operator`, `RNNOp`, `NetOp` has different logic for `DebugPrint`.

https://stackoverflow.com/questions/4571611/making-operator-virtual